app/views/layouts/application.html.erb

```diff
@@ -4,12 +4,54 @@
     <title>MembersOnly</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <%= stylesheet_link_tag "https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" %>
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body class="bg-dark"> 
+      <% if flash[:notice] %>
+        <div class="notification is-primary global-notification text-light text-center">
+          <p class="notice"><%= notice %></p>
+        </div>
+
+      <% end %>
+      <% if flash[:alert] %>
+        <div class="notification is-danger global-notification text-danger text-center">
+          <p class='alert'><%= alert %></p>
+        </div>
+
+      <% end %>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <a class="navbar-brand" href="/">Members Only!</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+          <ul class="navbar-nav">
+            <% if usuario_signed_in?%>
+            <li class="nav-item">
+              <a class="nav-link text-dark" href="/posts/new">New Post</a>
+            </li>
+            
+            <li class="nav-item ">                        
+              <%= link_to "Logout", destroy_usuario_session_path, method: :delete, class:"nav-link text-dark"%>  
+            </li>
+            <li class="nav-item">
+              <%= link_to  "Edit", edit_usuario_registration_path, class: "nav-link text-dark" %>
+            </li>  
+            <% else %>
+            <li class="nav-item ">            
+               <a class="nav-link text-dark" href="/usuarios/sign_in">Sing in</a>               
+            </li>
+            <li class="nav-item">
+              
+              <a class="nav-link text-dark" href="/usuarios/sign_up">Sing up<span class="sr-only">(current)</span></a> 
+            </li>
+            <% end %>
+          </ul>
```

Comment:
Please move logic into view helpers

Reply:
This is not addressed!!!! We can create a method for our login in our helpers and call it in our views